docs/translations/translations.json

```
@@ -75,6 +75,12 @@
   "resetDemo": "Reset demo",
   "resetDensity": "Reset density",
   "resetFocus": "Reset focus to test keyboard navigation",
+  "searchIcons": {
+    "clickToCopy": "Click to copy",
+    "close": "Close",
+    "copied": "Copied",
```

I think that these 3 labels should be generic (not scoped under `searchIcons`)

```suggestion
    "clickToCopy": "Click to copy",
    "close": "Close",
    "copied": "Copied",
```

---

I originally thought the same, but since they will dedupe on CrowdIn, there's no overhead. Doing it this way reduces the chance of accidentally removing a translation in use on another page. It should also make it easier to spot dead.